modules/fb_instant_articles_display/includes/view_mode_layout.inc

```diff
@@ -15,28 +16,28 @@ function fb_instant_articles_layout_form(&$form, &$form_state) {
   $view_mode = $form['#view_mode'];
   $form['#export_id'] = $entity_type . '|' . $bundle_name . '|' . $view_mode;
 
-  $layout = fb_instant_articles_get_node_layout_settings($bundle_name);
+  $layout = fb_instant_articles_display_get_node_layout_settings($bundle_name);
   $form['#fbia_layout'] = $layout;
 
-  fb_instant_articles_field_ui_fields($entity_type, $bundle_name, $view_mode, $form, $form_state);
+  fb_instant_articles_display_field_ui_fields($entity_type, $bundle_name, $view_mode, $form, $form_state);
 
   // Attach js.
-  $form['#attached']['js'][] = drupal_get_path('module', 'fb_instant_articles') . '/js/admin.js';
+  $form['#attached']['js'][] = drupal_get_path('module', 'fb_instant_articles_display') . '/js/admin.js';
```

Comment:
There is a JS error on the Manage Display page for the Facebook Instant Articles view mode.  Could be due to script ordering?  Digging in.